enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.stories.tsx

```diff
@@ -35,6 +35,34 @@ export default {
       control: { type: "radio" },
     },
     questionId: questionIdArgType,
+    entityTypes: {
+      options: [
+        "model",
+        "table",
+        "model_table",
+        "empty",
+        "null",
+        "undefined",
+        "invalid",
+      ],
+      mapping: {
```

Comment:
I just know we can map complex values (arry in this case) in storybook now. That's pretty neat.